### PR TITLE
added all necessary services for working with web

### PR DESCRIPTION
**Pull request by Den-moroz**

### CODE REVIEW

**Review by satori11111 — APPROVED** [review]
Perfect, if mentor approved previous task you can rebase it from development to main according to [doc](https://mate-academy.github.io/jv-program-fulltime/07_spring_boot/git-strategy)

File: src/main/java/core/basesyntax/bookstore/config/MapperConfig.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.bookstore.config;
+
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.NullValueCheckStrategy;
+
+@org.mapstruct.MapperConfig(
```
Comment (nacenik) [inline src/main/java/core/basesyntax/bookstore/config/MapperConfig.java:6]: You can import this annotation in the import statement 
```suggestion
@MapperConfig(
```

File: src/main/java/core/basesyntax/bookstore/controller/BookController.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(value = "/books")
```
Comment (nacenik) [inline src/main/java/core/basesyntax/bookstore/controller/BookController.java]: I'd recommend you add "API" before your endpoints. It shows that your application is a REST
```suggestion
@RequestMapping(value = "/api/books")
```

**Review by nacenik — CHANGES_REQUESTED** [review]
Good solution) Let's improve it a little bit) Also, resolve merge conflicts.

**Review by andrii-hoienko — APPROVED** [review]
Good

**Pull request merged**